.github/workflows/static/appspec.yml

```diff
@@ -0,0 +1,27 @@
+version: 0.0
+os: linux
+files:
+  - source: /files/etc/app_protect/conf/AppProtectDefaultPolicy.json
+    destination: /etc/app_protect/conf/
+  - source: /files/etc/app_protect/conf/log_all.json
+    destination: /etc/app_protect/conf/
+  - source: /files/etc/nginx/nginx.conf
+    destination: /etc/nginx/
+  - source: /files/etc/nginx/conf.d/
+    destination: /etc/nginx/conf.d/
+  - source: /scripts/index.html
```

Comment:
Seems like the path you have here doesn't match one for actual file: .github/workflows/static/index.html. I suspect the file won't be copied to instances and test will fail.

Reply:
We can not give the path .github/workflows/static/index.html. in appspec file, why because after the cft stack creation it will create a initial repository with stack name under code commit where in which default appspec file is available.

We have already discussed this that we need to have the customized files(appspec.yml and index.html) under scripts folder then only we can clone this repo and move the same after stack creation.

Reply:
Why don't you just copy index.html exactly like you copy appspec.yaml? (line39 oh the workflow) In this case there is no need to put it into initial repo.